modules/sql-engine/src/integrationTest/sql/subquery/scalar/test_grouped_correlated_subquery.test

```diff
@@ -1,6 +1,8 @@
 # name: test/sql/subquery/scalar/test_grouped_correlated_subquery.test
 # description: Test correlated subquery with grouping columns
 # group: [scalar]
+# Ignore https://issues.apache.org/jira/browse/IGNITE-22260
```

Comment:
we already mention these tickets for particular tests, no any reasons to repeat it here

Reply:
you fixed not for all cases